bin/adam-pyspark

```diff
@@ -23,10 +23,6 @@ ADAM_REPO="$(cd `dirname $0`/..; pwd)"
 CLASSPATH=$("$ADAM_REPO"/bin/compute-adam-classpath.sh)
 ADAM_JARS=$("$ADAM_REPO"/bin/compute-adam-jars.sh)
 
-if [ ! -z "$ADDL_JARS" ]; then
-  ADAM_JARS="$ADAM_JARS","$ADDL_JARS"
-fi
```

Comment:
Not certain.  Perhaps it was becoming unwieldy to keep track of where configs were coming from?

Eitherway, perhaps you could run your command like so:

```
export MY_SPARK_OPTS=...
adam-submit $MY_SPARK_OPTS -- $MY_ADAM_OPTS
```

Note that this only really affects `adam-submit`, as the interactive shells don't take any ADAM-specific options, AFAIK.